silkworm/rpc/json/types.cpp

```diff
@@ -226,7 +226,19 @@ void to_json(nlohmann::json& json, const BlockHeader& header) {
     } else {
         json["baseFeePerGas"] = nullptr;
     }
-    json["withdrawalsRoot"] = nullptr;  // waiting EIP-4895
+    json["AuRaSeal"] = nullptr;
+    json["AuRaStep"] = 0;
+    json["Verkle"] = false;
+    json["VerkleKeyVals"] = nullptr;
+    json["VerkleProof"] = nullptr;
```

Comment:
I suggest putting the following fields under Erigon JSON-RPC compatibility flag:

- AuRaSeal
- AuRaStep
- Verkle
- VrkleKeyVals
- VerkleProof